### ISSUE TITLE
The 'reviews update' code is currently disabled

### ISSUE THREAD

**Issue by audiodude** [openzim/wp1#19]
The old bot had code for updating 'reviews' as part of the update process. This code has not been migrated to the new bot and is currently not running.

It is unclear what the impact of this is, if any.

**Comment by audiodude** [comment]
It looks like this and #18 are both related to a table that kept track of whether certain articles were in an releases/reviews category, and the timestamp

**Comment by kelson42** [comment]
@walkerma Do you know why we had this feature? What was it doing in detail? Do you think this is a problem to get rid of it?

**Comment by walkerma** [comment]
Whilst I never saw the details of the code, I think the "Release" feature was used to keep track of whether a specific article was included in a release, such as WP 0.5 or WP 0.7.  Before, when you generated a list such as https://tools.wmflabs.org/enwp10/cgi-bin/list2.fcgi?run=yes&projecta=Chemistry&namespace=&pagename=&quality=&importance=&score=&showExternal=on&limit=500&offset=1&sorta=Score&sortb=Quality&filterRelease=on&reviewFilter=0&releaseFilter=0 you would get an entry in that table/list indicating 0.5, 0.7 etc.  I'm not sure if it ever worked for 0.8, though.  That way, it was easy for WikiProjects, and us, to see if there was a high-scoring article that needed to be included on the next release.  For the future this is probably not so important, because I would judge that instead of producing one "big" release for everyone to use such as 0.8, we (and others) will be producing hundreds of custom collections.  In principle it would be best to keep track of what articles get used in what releases, but that would involve a major revision anyway (e.g., you can't have the table showing 100 different collections next to every entry in the table!)

As for the review thing, I see that in the table it seems to show if something is GA or FA.  It's possible it did something different originally, such as indicate if an article was undergoing peer review.  Given that such information is usually shown anyway under Quality, I think that parameter can probably be dropped for future versions of the Web tool.

In summary: The Release parameter is useful (if greatly revised) but certainly not essential, and you can skip over it for now.  The Review parameter, as far as I can judge, is now largely obsolete.
